pkg/providers/instancetype/types.go

```diff
@@ -151,6 +152,10 @@ func computeRequirements(info *ec2.InstanceTypeInfo, offerings cloudprovider.Off
 	if info.ProcessorInfo != nil {
 		requirements.Get(v1beta1.LabelInstanceCPUManufacturer).Insert(lowerKabobCase(aws.StringValue(info.ProcessorInfo.Manufacturer)))
 	}
+	// EBS Max Bandwidth
+	if info.EbsInfo != nil && aws.StringValue(info.EbsInfo.EbsOptimizedSupport) != ec2.EbsOptimizedSupportUnsupported {
```

Comment:
Just one more thought here, for correctness, in case there's another setting included in the future, can we just check if it is supported? 
```suggestion
	if info.EbsInfo != nil && aws.StringValue(info.EbsInfo.EbsOptimizedSupport) != ec2.EbsOptimizedSupportSupported {
```

Reply:
We'll need to support both `ec2.EbsOptimizedSupportSupported` and `ec2.EbsOptimizedSupportDefault` values, as they both mean that `MaximumBandwidthInMbps` is available.

It might be better to use `ec2.EbsOptimizedSupportDefault` as Karpenter can't enable EBS Optimized for instances that have `ec2.EbsOptimizedSupportSupported` value, see #5053.

Reply:
Considering the list of instances that aren't EBS optimized by default are older generations, I believe the EC2 team has just recommended not having the option available, since they plan to just enable it by default on future generations that would support it. 

I'm personally comfortable just doing `ec2.EbsOptimizedSupportDefault` since, like you said, Karpenter doesn't support the ones that have it supported but not on out-of-the-box and I'm not really sure that there's a path where we would enable this.